UnitsNet/QuantityInfo.cs

```diff
@@ -21,52 +20,28 @@ namespace UnitsNet
     /// </remarks>
     public class QuantityInfo
     {
-        private static readonly string UnitEnumNamespace = typeof(LengthUnit).Namespace;
-
-        private static readonly Type[] UnitEnumTypes = typeof(Length)
-            .Wrap()
-            .Assembly
-            .GetExportedTypes()
-            .Where(t => t.Wrap().IsEnum && t.Namespace == UnitEnumNamespace && t.Name.EndsWith("Unit"))
-            .ToArray();
-
-        /// <summary>
-        ///     Constructs an instance.
-        /// </summary>
-        /// <param name="quantityType">The quantity enum value.</param>
-        /// <param name="unitInfos">The information about the units for this quantity.</param>
-        /// <param name="baseUnit">The base unit enum value.</param>
-        /// <param name="zero">The zero quantity.</param>
-        /// <param name="baseDimensions">The base dimensions of the quantity.</param>
-        /// <exception cref="ArgumentException">Quantity type can not be undefined.</exception>
-        /// <exception cref="ArgumentNullException">If units -or- baseUnit -or- zero -or- baseDimensions is null.</exception>
-        [Obsolete("QuantityType will be removed in the future. Use QuantityInfo(string, UnitInfo[], Enum, IQuantity, BaseDimensions) instead.")]
-        public QuantityInfo(QuantityType quantityType, [NotNull] UnitInfo[] unitInfos, [NotNull] Enum baseUnit, [NotNull] IQuantity zero, [NotNull] BaseDimensions baseDimensions)
```

Comment:
We can't remove this constructor, it would be a breaking change since it's been around for a long time.

Whenever me or another maintainer finds the time and energy to do a major version bump, then we can remove all the obsolete code.

Reply:
In theory it is a breaking change. But this constructor did not work for custom quantities and why should any external code manually create a QuantityInfo instance for one of the internal quantities? But I will restore the removed constructors.